index.html

```diff
@@ -167,14 +167,16 @@ <h2 class="languages-title">Skills</h2>
                 </section>
             </section>
             <section class="final-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</section>
-            <section class="data">
-                <input class="input-name" type="text" placeholder="Full name">
-                <input class="email-address" type="email" placeholder="Email address">
-                <textarea name="text1" class="text1" cols="30" rows="10">Enter text here</textarea>
-                <section class="side-im">
-                    <img class="side-form" src="images/form.svg" alt="side-form">
-                </section>
-                <button class="get-my-resume">Get in touch</button>
+            <section class="data">   
+                <form action="https://formspree.io/f/xayvoepp" method="post">
+                    <input class="input-name" type="text" placeholder="Full name" maxlength="30" required>
+                    <input class="email-address" type="email" placeholder="Email address" required>
+                    <textarea name="text1" class="text1" maxlength="500" cols="30" rows="10" required>Enter text here</textarea>
+                    <section class="side-im">
+                        <img class="side-form" src="images/form.svg" alt="side-form">
```

Comment:
- [ ] Let's make sure we use `background-image` property in CSS instead of putting it as img since it's not part of content it's just here for styling. also Let's make sure it's aligned to the right  to match Figma design

Your implementation | Figma Design
:------:|:-----:
![image](https://user-images.githubusercontent.com/87689547/154105243-b7090f4c-a5f3-43d2-98d8-388dc13a183a.png) | ![image](https://user-images.githubusercontent.com/87689547/154105311-28333106-6d22-4d9c-8a34-8dd0b8249674.png)